### PR TITLE
[Ready for Merge] Relic support

### PR DESCRIPTION
**Pull request by aydjay**
Added in support for Relics.

This is untested as I don't have any relics yet.

For work item in #667

### CODE REVIEW

**Comment by aydjay** [comment]
Having examined further - Item.cs getRarity will have to be updated. I can only assume they will increment the frameType value to indicate it's a relic. Will wait and see.

**Comment by aydjay** [comment]
I want to wait until I have a relic - or a bin file with a relic in it before the gets merged.

I suspect there will be a change to make before this gets commited to master.

File: Procurement/Controls/ItemHoverImage.xaml [hunk 1]
```diff
@@ -5,14 +5,10 @@
...
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:v="clr-namespace:Procurement.View"
-             xmlns:vm="clr-namespace:Procurement.ViewModel"
-             d:DesignHeight="150"
-             d:DesignWidth="150"
              mc:Ignorable="d">
 
     <UserControl.Resources>
         <v:VisibilityConverter x:Key="VisibilityConverter" />
-        <v:ItemToColorBrushConverter x:Key="vc" />
```
Comment (Stickymaddness) [inline Procurement/Controls/ItemHoverImage.xaml]: Why is this converter being removed?

Comment (aydjay) [inline Procurement/Controls/ItemHoverImage.xaml]: It's not used anywhere in the user control.

**Comment by aydjay** [comment]
image

Above is how the item hover for relic's render.

I checked that everything work with regards to relics.

Shame I didn't get a Kaoms as my first drop.

I'm happy for this to be merged.

**Comment by Stickymaddness** [comment]
Nice work, this looks great!

**Pull request merged**